Formula/pacmc.rb

```diff
@@ -0,0 +1,20 @@
+class Pacmc < Formula
+  desc "Minecraft package manager and launcher"
+  homepage "https://github.com/jakobkmar/pacmc"
+  url "https://github.com/jakobkmar/pacmc/releases/download/0.5.2/pacmc-0.5.2.tar"
+  sha256 "b0f4d338779acfb4a8898799beb545beb0a86ce9df19709765a871e33e7f5191"
+  license "AGPL-3.0-or-later"
+
+  depends_on "openjdk"
+
+  def install
+    rm_f Dir["bin/*.bat"]
+    libexec.install %w[bin lib]
+    env = Language::Java.overridable_java_home_env
+    (bin/"pacmc").write_env_script libexec/"bin/pacmc", env
+  end
+
+  test do
+    assert_match(/Hello user! pacmc works./, shell_output("#{bin}/pacmc debug test"))
```

Comment:
There is, but this way the test would depend on one of the third party mod repositories.

Reply:
How about running the search command from: https://github.com/jakobkmar/pacmc#quick-start

Reply:
That would work if the test just checks for the header returned by that command or something like that, however if the test checks for a package being listed by that command it would still depend on several external factors.
So should I just check for the search header? If so, there is no big difference to the `debug test` command, I guess.

Reply:
Unless the external factors are highly unreliable that's fine. The main difference is that we're testing something that's actually commonly used so more likely to show issues.

Reply:
That seems fine